src/components/Explore/utils.ts

```diff
@@ -75,9 +132,26 @@ function getDatasetIdByMetric(metric: ExploreMetric): DatasetId {
       return 'smoothedHospitalizations';
     case ExploreMetric.ICU_HOSPITALIZATIONS:
       return 'smoothedICUHospitalizations';
+    case ExploreMetric.VACCINATIONS_FIRST_DOSE:
+      return 'vaccinations';
+    case ExploreMetric.VACCINATIONS_COMPLETED:
+      return 'vaccinationsCompleted';
+    case ExploreMetric.ICU_USED:
+      return 'icuUtilization';
+    case ExploreMetric.POSITIVITY_RATE:
+      return 'testPositiveRate';
   }
 }
 
+export const getYFormat = (dataMeasure: DataMeasure, places: number) => {
+  const yFormat = (value: number) =>
+    dataMeasure === DataMeasure.PERCENT
+      ? formatPercent(value, places).toString()
+      : formatDecimal(value, places).toString();
```

Comment:
`formatPercent` and `formatDecimal` return string already, I don't think we need to convert them again

Reply:
oh true, not sure what made me convert it. removing!